### DISCUSSION TITLE
do the ip adresses specified in host lists need to be correct?

### DISCUSSION THREAD

**Discussion by Moth-Tolias** [evilsocket/opensnitch#1012] [Q&A] [answered]
that is to say - in an allow rule with "to this list of domains" checked, do the ip addresses in the list have any effect, or are they ignored?

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
Hi @Moth-Tolias ,

There can't be IP addresses in a host list. The list must be in hosts format: https://github.com/evilsocket/opensnitch/wiki/block-lists#lists-of-domains
The hosts lists use the format "IP domain" to specify a domain to block, where the IP is usually 0.0.0.0 or 127.0.0.1. If we don't find those IPs, the domain of that line is excluded from the list.

On the other hand, only the domains of the list are loaded, to be compared against the domain of outbound connections.

**Reply by Moth-Tolias** [reply]
i think i understand. if the ip address is neither `0.0.0.0` nor `127.0.0.1`, then the entire line is ignored.
other than that, they have no effect, and only the domains are used?

**Reply by gustavo-iniguez-goya** [reply]
yes, that's it. Also if the domain is one of "local", "localhost", "localhost.localdomain", "broadcasthost" or "127.0.0.1" the entry is excluded.
This is mainly to support public lists of domains to block in "hosts" format.
